lambda/custom-nft/contracts/contracts/MetaKeepNFT.sol

```diff
@@ -0,0 +1,110 @@
+//SPDX-License-Identifier: UNLICENSED
+
+// Solidity files have to start with this pragma.
+// It will be used by the Solidity compiler to validate its version.
+pragma solidity ^0.8.9;
+
+// We import this library to be able to use console.log
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
@rohan-passbird here we are inherting the OZ's Interface to make calls from this contract. Ideally it should be metakeep's ERC20 Interface.